main.go

```diff
@@ -30,11 +30,17 @@ import (
 	"github.com/google/uuid"
 )
 
-var xdsServerUri = flag.String("xds-server-uri", "trafficdirector.googleapis.com:443", "override of server uri, for testing")
-var outputName = flag.String("output", "-", "output file name")
-var gcpProjectNumber = flag.Int64("gcp-project-number", 0,
-	"the gcp project number. If unknown, can be found via 'gcloud projects list'")
-var vpcNetworkName = flag.String("vpc-network-name", "default", "VPC network name")
+var (
+	xdsServerUri     = flag.String("xds-server-uri", "trafficdirector.googleapis.com:443", "override of server uri, for testing")
+	outputName       = flag.String("output", "-", "output file name")
+	gcpProjectNumber = flag.Int64("gcp-project-number", 0,
+		"the gcp project number. If unknown, can be found via 'gcloud projects list'")
+	vpcNetworkName          = flag.String("vpc-network-name", "default", "VPC network name")
+	enableFileWatcherConfig = flag.Bool("enable-file-watcher-config", false, "whether or not to generate file_watcher certificate provider config")
```

Comment:
I think we should add EXPERIMENTAL or something to the description. We'll want to remove this flag in the future.

Reply:
Done.

Reply:
We need experimental where the user would see. This comment is useless. I was suggesting in the description string.